### PR TITLE
Added word2vec script and sample data

### PR DESCRIPTION
**Pull request by ricsuh**

### CODE REVIEW

**Comment by cgreene** [comment]
The first PR! 🎉

File: word2vec.py
Comment (danich1) [inline word2vec.py]:
```suggestion
        for line in open(self.filename, 'r'):
```
It is good to be explicit on the file mode just so you don't accidentally open a file in writing mode and undo everything.

File: word2vec.py
Comment (danich1) [inline word2vec.py]: Would make your code easier to read if you convert long lines into lines with spaces:
```python
do_mesh_pairs = dict(zip(
     do_mesh_df.doid_code,
     "MESH:"+do_mesh_df.mesh_id
))
```

File: word2vec.py
Comment (danich1) [inline word2vec.py]: Side note. You can also do this (dont have to change but just a pandas tip):

```python
gene_disease_df = gene_disease_df.query("~mesh_id.str.contains('DOID:')")
```

File: word2vec.py
Comment (danich1) [inline word2vec.py]: White space tip:

```python
positive_pairs = (
   gene_disease_df
   [["mesh_id", "entrez_gene_id"]]
   .values
   .tolist()
)
```

File: word2vec.py
Comment (danich1) [inline word2vec.py]: Change to:

```python
gene_disease_df["random_gene"] = (
   random.sample(
      gene_disease_df["entrez_gene_id"].values.tolist(),
       len(gene_disease_df["entrez_gene_id"].values.tolist())
   )
)

File: word2vec.py
Comment (danich1) [inline word2vec.py]: Change to:

```python
new_row = {
      "disease": pair[0], 
      "gene": pair[1],
      "class": pair[2], 
      "score": score
}
```
```

File: word2vec.py
Comment (danich1) [inline word2vec.py]: White space change as above.

**Review by danich1 — APPROVED** [review]
Congrats on your first pull request. 🎉 🎉 

I approve provided that you make the suggested changes above before merging!

**Pull request merged**
